services/lambda/src/it/java/software/amazon/awssdk/services/lambda/IntegrationTestBase.java

```diff
@@ -91,24 +91,45 @@ public static void tearDown() {
     }
 
     private static File setupFunctionZip(String jsFile) throws IOException {
-        InputStream in = IntegrationTestBase.class.getResourceAsStream(jsFile);
+        try {
```

Comment:
You should use try with resource:
```
try(InputStream in = IntegrationTestBase.class.getResourceAsStream(jsFile)){
...
}
```
Thanks to that you don't need to deal with closing, it implicitly will create a finally and clode the stream